src/main/main.cpp

```diff
@@ -153,5 +212,6 @@ void loop()
 		statusUpdate(); // report status update
 		sdcard->writeToSDCard();
     buzzer->whichAlertToMake(tempStatus, humStatus); // Check if noise should be made
+    statusUpdate();									 // report status update
```

Comment:
How come this line is here as well as on line 212? it seems to be calling the statusUpdate() method twice